### ISSUE TITLE
XMR-stak mining not consistency

### ISSUE THREAD

**Issue by remark23** [fireice-uk/xmr-stak#1399]
I mining AEON on: 

  - AMD FX-8320 overclocked on 4.4 Ghz.
  - 32 GB RAM
  - No GPU mining
  - Linux OS

For some time, about 30 seconds minning speed is ~1.3kH/s, then the speed falls on ~400H/s for about 30 seconds, and so it repeats..
With "sensors" command I see that CPU power is 110W for some time, well then 35W for some time..
As if the processor turns on and off alternately. It's like the processor activates some protection, CPU temperature is max 50-51C.

1. At one point shows:

HASHRATE REPORT - CPU
| ID |    10s |    60s |    15m | ID |    10s |    60s |    15m |
|  0 |  338.1 |  240.1 |  227.7 |  1 |  338.9 |  240.9 |  228.3 |
|  2 |  338.5 |  240.3 |  228.1 |  3 |  338.2 |  240.5 |  228.3 |
Totals (CPU):  1353.7  961.8  912.3 H/s
-----------------------------------------------------------------
Totals (ALL):   1353.7  961.8  912.3 H/s
Highest:  1361.3 H/s

2. At second point shows:

HASHRATE REPORT - CPU
| ID |    10s |    60s |    15m | ID |    10s |    60s |    15m |
|  0 |  106.3 |  218.4 |  228.8 |  1 |  106.6 |  219.2 |  229.7 |
|  2 |  106.5 |  219.2 |  229.6 |  3 |  106.7 |  219.0 |  229.5 |
Totals (CPU):   426.2  875.7  917.6 H/s
-----------------------------------------------------------------
Totals (ALL):    426.2  875.7  917.6 H/s
Highest:  1361.3 H/s

**Comment by remark23** [comment]
No Laptop, desktop PC with a lot of fans, CPU temperature does not exceed 50-51C.

**Comment by Spudz76** [comment]
What is motherboard make/model?
You may also find `turbostat` utility useful to see what is going on with clocks per core and such.

**Comment by remark23** [comment]
Motherboard model: Gigabyte GA-970A-DS3.

Turbostat report:

turbostat version 17.04.12 - Len Brown <lenb@kernel.org>
CPUID(0): AuthenticAMD 13 CPUID levels; family:model:stepping 0xf:2:0 (15:2:0)
CPUID(1): SSE3 MONITOR - - - TSC MSR - -
CPUID(6): APERF, No-TURBO, No-DTS, No-PTM, No-HWP, No-HWPnotify, No-HWPwindow, No-HWPepp, No-HWPpkg, No-EPB
turbostat: cpu3: msr offset 0x1a0 read failed: Input/output error

**Comment by Spudz76** [comment]
Having read most of that thread and the DS3 info, that CPU is not supported likely because it does what it's doing.  You may be able to add-on extra cooling for the VRMs however they still aren't the bigger phases.  Or, run a stable speed by underclocking to target 110W or less.  Or find a FX-83xxE which are 95W to stay within the VRM usefulness.  Or force some cores into powersave and don't use them, with 8MB of cache you probably hit ceiling at half the cores anyway.

**Comment by remark23** [comment]
Thnx!
All right, I'll try to lower the CPU clock so I can see what's going to happen..

**Comment by remark23** [comment]
The problem was in the following:

**The VRM module was overheated.**
Previously, a standard CPU cooler was mounted that directs the air to the motherboard and cools the VRM, but the processor was overheating.
The CPU cooler has been replaced with a standing big cooler, that directs the air horizontally to the motherboard so that the air does not reach the VRM module.
I added another fan to direct the air to VRM and it's okay now.

Thanks for the help because I did not know that the VRM could make a problem.
Best regards.
